packages/common-all/src/dnode.ts

```diff
@@ -1262,8 +1262,13 @@ export class SchemaUtils {
       }
       const tempNoteProps = _.pick(tempNote, this.TEMPLATE_COPY_PROPS);
       _.forEach(tempNoteProps, (v, k) => {
-        // @ts-ignore
-        note[k] = v;
+        // If note body exists, append template's body instead of overriding
```

Comment:
instead of introducing an if statement here, can we remove `body` from `TEMPLATE_COPY_PROPS` and add the logic outside of the for loop?

Reply:
Sure that can be done. I'll still need to keep the if logic to check if body exists since that determines whether we add a \n or not. That work?